R/gg-plots.R

```diff
@@ -33,7 +18,7 @@ eval_data_col <- function(data, aes_col) {
 #' mapping <- ggplot2::aes(Petal.Length)
 #' mapping_string(mapping$x)
 mapping_string <- function(aes_col) {
-  gsub("^~", "", deparse(aes_col, 500L))
+  gsub("^~(?:\\.data[$])?", "", deparse(aes_col, 500L))
```

Comment:
this is somewhat sketchy, I'm not exactly sure if this is the best fix for the newly observed issue that `data$` becomes part of the `mapping_string()` output.